index.js

```diff
@@ -0,0 +1,83 @@
+import axios from 'axios'
+
+import {
+    getInput,
+    setFailed,
+    getMultilineInput,
+    setOutput,
+} from '@actions/core'
+import { context } from '@actions/github'
+
+const appId = getInput('app-id')
+const workflowId = getInput('workflow-id')
+const token = getInput('token')
+
+const labels = getMultilineInput('labels')
+
+let branch = getInput('branch')
+let tag = getInput('tag')
+
+if (!branch && !tag) {
```

Comment:
so do we send 2 values to API?

Reply:
Tag has precedence over branch in Codemagic builds

If both variables are missing, we need to resolve them from GitHub Actions event

Reply:
> Tag has precedence over branch in Codemagic builds

let's add it to description if user defines tag and branch at the same moment